web-registry/src/globals.ts

```diff
@@ -1,9 +0,0 @@
-import {
-  URL_REGISTRY_MARKETPLACE_LEGAL,
-  URL_REGISTRY_TERMS_SERVICE,
-  URL_WEB_PRIVACY,
-} from 'web-components/lib/utils/globals';
-
-export const URL_PRIVACY = URL_WEB_PRIVACY;
-export const URL_TERMS_SERVICE = URL_REGISTRY_TERMS_SERVICE;
-export const URL_MARKETPLACE_LEGAL = URL_REGISTRY_MARKETPLACE_LEGAL;
```

Comment:
Not directly related by this PR but these constants should have been created in registry in the first place.
It feels weird to import this kind of data from our component library.

Reply:
Yeah, I had that in mind too. I thought that one of these global variables was being used on the website as well. I just checked that it's not, so I make this change of bringing the global variables to the registry

Reply:
@flagrede done! Had to move a few components to the registry also